src/main/java/uk/co/nazire/service/StudentService.java

```diff
@@ -0,0 +1,13 @@
+package uk.co.nazire.service;
+
+import java.util.List;
+
+import uk.co.nazire.model.Student;
+
+public interface StudentService {
+	
+	List<Student> getStudentList();
+	
+	Student getId(Long id);
+
```

Comment:
Be carefully for code style

Reply:
I have done. I should be carefully after that.